ui/src/mixins/MobileLayout.js

```diff
@@ -0,0 +1,209 @@
+            var fontSize='';
+            switch (screenSize) {
+              case 'xs': fontSize = '12px'
+                break;
+              case 'sm': fontSize =  '12px'
+                break;
+              case 'md': fontSize =  '14px'
+                break;
+              case 'lg': fontSize =  '16px'
+                break;
+              case 'xl': fontSize =  '16px'
+                break;
+            }
+            return fontSize;
+        },
+        gridLayoutShowRecord(screenSize){
+            var layout = "";
+            switch (screenSize) {
+              case 'xs': layout = 'xs12 pr-3 ml-2';
+                break;
+              case 'sm': layout = 'xs8 pr-3 ml-2';
+                break;
+              case 'md': layout = 'xs6 pr-3';
+                break;
+              case 'lg': layout = 'xs6 pr-3';
+                break;
+              case 'xl': layout = 'xs6 pr-3'
+            }
+            return layout;
+        },
+        gridLayoutSearchResult(screenSize){
+            var layout = "";
+            switch (screenSize) {
+              case 'xs': layout = 'xs12';
+                break;
+              case 'sm': layout = 'xs9';
+                break;
+              case 'md': layout = 'xs9';
+                break;
+              case 'lg': layout = 'xs9';
+                break;
+              case 'xl': layout = 'xs9'
+            }
+            return layout;
+        },
+        gridLayoutShowResult(screenSize){
+            var layout = "";
+            switch (screenSize) {
+              case 'xs': layout = 'xs12';
+                break;
+              case 'sm': layout = 'xs12';
+                break;
+              case 'md': layout = 'xs9';
+                break;
+              case 'lg': layout = 'xs9';
+                break;
+              case 'xl': layout = 'xs9'
+            }
+            return layout;
+        },
+    smallScreenCompute(screenSize){
+        var smallScreen=false;
+        if(screenSize == "xs")
+        {
+          smallScreen = true;
+        }
+        else{
+          smallScreen = false;
+        }
+        return smallScreen;
+    },
+    smallScreenSearchCompute(screenSize){
+        var smallScreen=false;
+        if(screenSize == "xs")
+        {
+          smallScreen = true;
+        }
+        else if(screenSize == "sm")
+        {
+          smallScreen = true;
+        }
+        else{
+          smallScreen = false;
+        }
+        return smallScreen;
+    },
+    detailCardTitleStyle(screenSize)
+    {
+        var style="";
+        switch (screenSize) {
+            case 'xs': style='title';
+            break;
+            case 'sm': style='title';
+            break;
+            case 'md': style='display-1';
+            break;
+            case 'lg': style='display-1';
+            break;
+            case 'xl': style='display-1';
+            break;
+        }
+        return style;
+    },
+    detailCardBtnSize(screenSize){
```

Comment:
If this is only for the detail card component, it shouldn't be in a mixin and just be in that component.